docs: add in the pull request file to not comment in closed prs (#7592)
typescript-eslint/typescript-eslint: docs/contributing/Pull_Requests.mdx

- Comment on an existing PR asking for updates
  - Reasoning: Your PR hasn't been forgotten! The volunteer maintainers have limited time to work on the project, and they will get to it as soon as they are able.
    - One exception: if a PR has been blocked on a question to a maintainer for 3 or more months, please ping us - we probably lost track of it.

> - Comment on a closed PR.

Reviewer: ```suggestion
- Comment on a closed PR
```

> - Comment on a closed PRs.

Reviewer: ```suggestion
- Comment on a closed PR.
```

>   - Reasoning: It is much easier for maintainers to not lose track of things if they are posted as issues. If you think there’s a bug in typescript-eslint, the right way to ask is to [file a new issue](https://github.com/typescript-eslint/typescript-eslint/issues/new/choose). The issue templates include helpful & necessary practices such as making sure you're on the latest version of all our packages.

Reviewer: ```suggestion
  - Reasoning: It is much easier for maintainers to not lose track of things if they are posted as issues. If you think there's a bug in typescript-eslint, the right way to ask is to [file a new issue](https://github.com/typescript-eslint/typescript-eslint/issues/new/choose). The issue templates include helpful & necessary practices such as making sure you're on the latest version of all our packages. You can provide the link to the relevant PR to add more context.
```

Author: Nice, thank you!

>   - Reasoning: It is much easier for maintainers to not lose track of things if they are posted as issues. If you think there’s a bug in typescript-eslint, the right way to ask is to [file a new issue](https://github.com/typescript-eslint/typescript-eslint/issues/new/choose). The issue templates include helpful & necessary practices such as making sure you're on the latest version of all our packages. 

Author: ```suggestion
  - Reasoning: It is much easier for maintainers to not lose track of things if they are posted as issues. If you think there’s a bug in typescript-eslint, the right way to ask is to [file a new issue](https://github.com/typescript-eslint/typescript-eslint/issues/new/choose). The issue templates include helpful & necessary practices such as making sure you're on the latest version of all our packages.
```
